src/components/body/results/__snapshots__/Results.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Results should render correctly 1`] = `ShallowWrapper {}`;
```

Comment:
As you can see your snapshot doesn't bring so match information about component)
You need to define snapshot serializer to 'ensyme-to-json' to handle this

Reply:
done